network-api/networkapi/views.py

```diff
@@ -19,3 +22,15 @@ def review_app_help_view(request):
         return render(request, "reviewapp-help.html")
     else:
         return HttpResponse(status=404)
+
+
+@require_GET
+def apple_pay_domain_association_view(request):
+    """
+    Returns string needed for Apple Pay domain association/verification.
+    """
+    if Site.find_for_request(request).hostname == "www.mozillafestival.org":
+        verification_key = settings.APPLE_PAY_DOMAIN_ASSOCIATION_KEY_MOZFEST
+    else:
+        verification_key = settings.APPLE_PAY_DOMAIN_ASSOCIATION_KEY_FOUNDATION
+    return HttpResponse(verification_key, content_type="text/plain")
```

Comment:
Considering that we can not guarantee in code that the environment variables are set, what do you think about responding with different error codes: 

* 400 Bad request - the site can not be matched to either of the cases (Foundation or Mozfest -- based on Site root page type)
* 501 Not implemented - no verification key can be determined while the site is valid (e.g. for the case where the environment variable is not set).

Reply:
Updated!